### PR TITLE
Fix closed wall runtime

### PR DESCRIPTION
**Pull request by Dennok**
## About The Pull Request

Sometimes after massive turf change, mostly on shuttle landing, some code try to update gas visuals of closed walls. This code make it at least dont cry runtime error.

## Changelog
:cl:
fix: Fixed some turf runtime error.
/:cl:

### CODE REVIEW

File: code/modules/atmospherics/environmental/LINDA_turf_tile.dm [hunk 1]
```diff
@@ -165,6 +167,7 @@
 
 	UNSETEMPTY(new_overlay_types)
 	src.atmos_overlay_types = new_overlay_types
+	return
```
Comment (SuperNovaa41) [inline code/modules/atmospherics/environmental/LINDA_turf_tile.dm:170]: What's the point of this return at the end?

**Comment by Kapu1178** [comment]
this isn't fixing the problem, it's obscuring it.

**Comment by SuperNovaa41** [comment]
He's right

**Comment by Dennok** [comment]
Yes, But atleaset it dont create runtime and dont break things. This is some asynchrony problems, and i has no idea how tu fix it, Besides removing of 'check tick' from shutle landing code.

File: code/modules/atmospherics/environmental/LINDA_turf_tile.dm (hunk 1 above)
Comment (Dennok) [reply code/modules/atmospherics/environmental/LINDA_turf_tile.dm:170]: Dont allow turf/open/update_visuals() acivate CRASH in turf/update_visuals() 
It must be activated when somethng try to update_visuals() on not open turfs

**Comment by Kapu1178** [comment]
If you don't know how to fix it, leave it there so someone who does, does. Obscuring the problem only makes it worse.

**Comment by Dennok** [comment]
It anyway throw crashes. 
but no runtimes

>leave it there so someone who does, does

I leave it in TG near year or 2 back

**Comment by SuperNovaa41** [comment]
I would rather have a full fix than a half fix

**Pull request closed without merging**